core/src/main/java/org/elasticsearch/index/translog/TranslogReader.java

```diff
@@ -41,28 +41,45 @@
  * an immutable translog filereader
  */
 public class TranslogReader extends BaseTranslogReader implements Closeable {
+
     private static final byte LUCENE_CODEC_HEADER_BYTE = 0x3f;
     private static final byte UNVERSIONED_TRANSLOG_HEADER_BYTE = 0x00;
 
-
-    private final int totalOperations;
     protected final long length;
+    private final int totalOperations;
```

Comment:
I'm not sure it's worth having these extra fields now that we have a checkpoint? code that needs it can either call the getter methods or access the checkpoint directly?